src/deploy/RPM_build/noobaa.spec

```diff
@@ -95,6 +95,7 @@ ln -s /usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.conf $RPM_BU
 %{_unitdir}/noobaa.service
 /etc/logrotate.d/noobaa/logrotate_noobaa.conf
 /etc/rsyslog.d/noobaa_syslog.conf
+/usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.sh
```

Comment:
is it really needed to specify? all the files in /usr/local/noobaa-core are part of the RPM...